deepwell-core/src/models/page.rs

```diff
@@ -78,3 +78,16 @@ impl Page {
         self.deleted_at.is_none()
     }
 }
+
+impl<'a> From<(Page, Votes)> for ftml::PageInfo<'a> {
+    fn from(p: (Page, Votes)) -> Self {
```

Comment:
Could be more readable if deconstructed:
```suggestion
    fn from((page, votes): (Page, Votes)) -> Self {
```
then you can do `title: page.title()` instead of `p.0` and `p.1`